### PR TITLE
Remove duplicate functions

### PR DESCRIPTION
**Pull request by olejandro**

### CODE REVIEW

**Review by siddharth-krishna — APPROVED** [review]

File: times_reader/transforms.py [hunk 1]
```diff
@@ -2174,6 +2173,7 @@ def convert_aliases(input: Dict[str, DataFrame]) -> Dict[str, DataFrame]:
     return output
 
 
+# TODO: A function like this is also inluded in main.py. Should we delete one?
```
Comment (olejandro) [inline times_reader/transforms.py:2176]: @siddharth-krishna @samwebster should we delete the function from here or from main?

File: times_reader/transforms.py [hunk 2]
```diff
@@ -2198,6 +2198,7 @@ def dump_tables(tables: List, filename: str) -> List:
     return tables
 
 
+# TODO: A function like this is also inluded in main.py. Should we delete one?
 def expand_rows_parallel(
     tables: List[datatypes.EmbeddedXlTable],
 ) -> List[datatypes.EmbeddedXlTable]:
```
Comment (olejandro) [inline times_reader/transforms.py:2204]: @siddharth-krishna @samwebster should we delete the function from here or from main?

Comment (siddharth-krishna) [reply times_reader/transforms.py:2204]: I don't know why there's two of these. My guess is it was copy-pasted and not moved by accident. I'd say remove this one from main.

File: times_reader/transforms.py (hunk 1 above)
Comment (siddharth-krishna) [reply times_reader/transforms.py:2176]: But for this one, perhaps keep it in main and remove from transforms? This is less a transformation and more of an IO action.
Sam, correct me if I'm missing something.

**Pull request merged**